### PR TITLE
MISUV-3437 Content update for Tax year summary

### PR DESCRIPTION
**Pull request by andriikondiuk**

### CODE REVIEW

File: app/views/TaxYearSummary.scala.html [hunk 1]
```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  *@
 
-@import implicits.ImplicitCurrencyFormatter._
+@import _root_.implicits.ImplicitCurrencyFormatter._
```
Comment (joesheridan) [inline app/views/TaxYearSummary.scala.html:17]: is this needed?

File: conf/messages [hunk 2]
```diff
@@ -146,15 +146,15 @@ taxYears.oldSa.agent.content.3								   = This will be a different Government G
 tax-year-summary.heading                                           = Tax year summary
 tax-year-summary.heading-secondary                                 = 6 April {0} to 5 April {1}
 tax-year-summary.calculation-date                                  = Calculation date
-tax-year-summary.calc-from-last-time                               = This is only based on figures we’ve already received to {0}. It’s not your final tax bill.
+tax-year-summary.calc-from-last-time                               = This is not your final tax bill, it''s only based on figures we''ve already received.
```
Comment (joesheridan) [inline conf/messages:149]: two single quotes ('')? is this right?

File: test/views/TaxYearSummaryViewSpec.scala [hunk 3]
```diff
@@ -181,15 +181,15 @@ class TaxYearSummaryViewSpec extends ViewSpec with FeatureSwitching {
     val estimate: String = s"6 April ${testYear - 1} to 1 January 2020 estimate"
     val totalDue: String = "Total Due"
     val taxDue: String = "£4.04"
-    val calcDateInfo: String = s"This is only based on figures we’ve already received to $calcDate. It’s not your final tax bill."
+    val calcDateInfo: String = "This is not your final tax bill, it's only based on figures we've already received."
```
Comment (joesheridan) [inline test/views/TaxYearSummaryViewSpec.scala:184]: we should get this from the messagesAPI ideally..

File: test/views/TaxYearSummaryViewSpec.scala [hunk 4]
```diff
@@ -181,15 +181,15 @@ class TaxYearSummaryViewSpec extends ViewSpec with FeatureSwitching {
...
     val calcEstimateInfo: String = "This is a year to date estimate based on figures we’ve already received."
     val taxCalculation: String = s"6 April ${testYear - 1} to 5 January $testYear calculation"
     val taxCalculationHeading: String = "Calculation"
     val taxCalculationTab: String = "Calculation"
     val taxCalculationNoData: String = "No calculation yet"
     val forecastNoData: String = "No forecast yet"
     val forecastNoDataNote: String = "You will be able to see your forecast for the whole year once you have sent an update."
-    val unattendedCalcPara: String = "! We have updated the calculation for you. You can see more details in your record-keeping software."
+    val unattendedCalcPara: String = "! We've updated the calculation for you. Check your record-keeping software for more details."
```
Comment (joesheridan) [inline test/views/TaxYearSummaryViewSpec.scala:192]: same

File: conf/messages [hunk 5]
```diff
@@ -146,15 +146,15 @@ taxYears.oldSa.agent.content.3								   = This will be a different Government G
...
 tax-year-summary.calc-estimate-info                                = This is a year to date estimate based on figures we’ve already received.
 tax-year-summary.total-due                                         = Total Due
 tax-year-summary.contents                                          = Contents
 tax-year-summary.tax-calculation.date                              = 6 April {0} to 5 January {1} calculation
 tax-year-summary.tax-calculation                                   = Calculation
 tax-year-summary.tax-calculation.no-calc                           = No calculation yet
 tax-year-summary.tax-calculation.no-calc.note                      = You will be able to see your latest tax year calculation here once you have sent an update.
-tax-year-summary.tax-calculation.unattended-calc                   = We have updated the calculation for you. You can see more details in your record-keeping software.
+tax-year-summary.tax-calculation.unattended-calc                   = We''ve updated the calculation for you. Check your record-keeping software for more details.
```
Comment (joesheridan) [inline conf/messages:157]: same with quotes

File: app/views/TaxYearSummary.scala.html (hunk 1 above)
Comment (andriikondiuk) [reply app/views/TaxYearSummary.scala.html:17]: It's only improve discoverability

File: conf/messages (hunk 5 above)
Comment (andriikondiuk) [reply conf/messages]: Thanks, Done

File: test/views/TaxYearSummaryViewSpec.scala (hunk 4 above)
Comment (andriikondiuk) [reply test/views/TaxYearSummaryViewSpec.scala]: Thanks, But we have a Tech Debt ticket to fix all of those messages in one go.

File: test/views/TaxYearSummaryViewSpec.scala (hunk 3 above)
Comment (andriikondiuk) [reply test/views/TaxYearSummaryViewSpec.scala]: Thanks, But we have a Tech Debt ticket to fix all of those messages in one go.

File: conf/messages (hunk 2 above)
Comment (andriikondiuk) [reply conf/messages]: Thank's, I did exactly as design team want it. Double tick will render to just one. But you are right I've changed it to what we are using. Done

**Pull request merged**